lib/routes/visionias/utils.ts

```diff
@@ -0,0 +1,107 @@
+import { getCurrentPath } from '@/utils/helpers';
+const __dirname = getCurrentPath(import.meta.url);
+
+import { DataItem } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import { art } from '@/utils/render';
+import path from 'node:path';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const baseUrl = 'https://visionias.in';
+
+export async function extractNews(item, selector) {
+    if (item.link === '') {
+        return item;
+    }
+    const response = await ofetch(item.link || '');
+    const $$ = load(response);
+    const content = $$(selector);
+    const heading = content.find('div.space-y-4 > h1').text();
+    const mainGroup = content.find('div.flex > div.w-full');
+    const postedDate = mainGroup.find('p:contains("Posted ") > strong').text();
+    const updatedDate = mainGroup.find('p:contains("Updated ") > strong').text();
```

Comment:
Use the content from `$('meta[property="article:published_time"]')` and `$('meta[property="article:modified_time"]')` instead